docs/api/Makefile

```diff
@@ -0,0 +1,12 @@
+# ls ../../nimbleclient/v1/api/*[a-z].py | xargs basename | sed 's#\.py$##' | tr \\n ' '
```

Comment:
nit: do we need this commented out code?

Reply:
I thought this would be handy to have around if we need to regenerate when we add new API endpoints.

Reply:
oh okay.. that should be fine then.. just comment the intend as well :)